Dockerfile-ModelServer

```diff
@@ -33,4 +33,9 @@ COPY --from=builder /code/dist/gordo-components-packed.tar.gz .
 # Install gordo-components, packaged from earlier 'python setup.py sdist'
 RUN pip install ./gordo-components-packed.tar.gz
 
+# Nonroot user for running CMD
+RUN groupadd -g 999 gordo && \
```

Comment:
why 999?

Reply:
We use 1000

Reply:
That was suggested in the Medium article you linked to. I figured he was using that as that would indicate the group/user is a dynamically generated system group/user.

Reply:
Should I use a 1000 as the uid/gid?

Reading [this](https://refspecs.linuxfoundation.org/LSB_3.0.0/LSB-Core-generic/LSB-Core-generic/uidrange.html), it seems that perhaps 499 may be better; though, I'm not sure what any real effect of doing one vs. the other could be.